cpp/oneapi/dal/algo/svm/test/batch.cpp

```diff
@@ -539,6 +540,7 @@ TEMPLATE_LIST_TEST_M(svm_batch_test,
                      "svm can classify any two labels",
                      "[svm][integration][batch][linear]",
                      svm_types) {
+    SKIP_IF(this->get_policy().is_cpu());
```

Comment:
Why this test is skipped on CPU? If it's temporal, corresponding comment should be added, `// TODO:`